src/twisted/python/test/test_bytes.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+Tests for L{twisted.python.bytes}.
+"""
+
+from __future__ import division, absolute_import, print_function
```

Comment:
just a minor comment... to make it easier to test py3 stuff on py27 I also import unicode_literals